src/fhirtypes/ElementDefinition.ts

```diff
@@ -215,12 +216,14 @@ export class ElementDefinition {
 
   /**
    * Get the StructureDefinition for ElementDefinition
-   * @param {ResolveFn} resolve - A function that can resolve a type to a StructureDefinition instance
+   * @param {Fishable} fisher - A fishable implementation for finding definitions and metadata
    * @returns {StructureDefinition} the StructureDefinition of ElementDefinition
    */
-  getOwnStructureDefinition(resolve: ResolveFn = () => undefined): StructureDefinition {
+  getOwnStructureDefinition(fisher: Fishable): StructureDefinition {
     if (this._edStructureDefinition == null) {
-      this._edStructureDefinition = resolve('ElementDefinition');
+      this._edStructureDefinition = StructureDefinition.fromJSON(
+        fisher.fishForFHIR('ElementDefinition', Type.Type)
+      );
```

Comment:
This might be a bit of a stretch, but if somehow there isn't anything named `ElementDefinition` available, `fromJSON` will receive `undefined` and throw a `TypeError`. While I recognize this is colossally unlikely to occur, is this the first place the processing would break when `ElementDefinition` doesn't exist?

Reply:
I'm not sure if this is the first place, but it would be a fatal error.  We absolutely need it.  Since it is part of the core FHIR spec, it's very unlikely it's not there.  The only case that might happen is if someone manually mucked with their `./~fhir` cache.  I think if they didn't have the FHIR core spec at all, we would fail way before this.  So... I'm not too concerned.